acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/passThrough/PassThroughAcceptanceTests.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.tests.passThrough;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static tech.pegasys.ethsigner.tests.dsl.Gas.GAS_PRICE;
+import static tech.pegasys.ethsigner.tests.dsl.Gas.INTRINSIC_GAS;
+
+import tech.pegasys.ethsigner.tests.AcceptanceTestBase;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+import org.junit.Test;
+import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.utils.Convert;
+
+public class PassThroughAcceptanceTests extends AcceptanceTestBase {
+
+  static final String TRANFER_AMOUNT_ETHER = "10";
+  static final BigInteger TRANSFER_AMOUNT_WEI =
+      Convert.toWei(TRANFER_AMOUNT_ETHER, Convert.Unit.ETHER).toBigIntegerExact();
+
+  @Test
+  public void ethGetTransactionCountReturnCorrectNumber() throws Exception {
+    final BigInteger startTransactionCount =
+        ethNode().transactions().count(richBenefactor().address());
```

Comment:
To tidy up the syntax, overload `count()` with another method that accepts an `Account`.
(similar to what was done for `balance`)

Reply:
done